contracts/collateral/OverlayV1OVLCollateral.sol

```diff
@@ -32,7 +31,7 @@ contract OverlayV1OVLCollateral is ERC1155Supply {
     Position.Info[] public positions;
 
     IOverlayV1Mothership public immutable mothership;
-    IOverlayTokenNew immutable public ovl;
+    IOverlayToken immutable public ovl;
```

Comment:
I don't understand the removal of the new token. The auditors found a bug in it that would not have been good. But not one of them chimed in about the pattern itself being a no go. 

I don't find a risk in this pattern. It's a highly specialized permissioned function that is only invoked from non manipulable immutable smart contracts. 

This is essentially the nature of our entire system. Our entire system does these things and I find no anxiety in collapsing some of the functions in this highly prescribed and narrow domain.